client-v2/src/shared/components/icons/Icons.tsx

```diff
@@ -28,7 +28,7 @@ const DownCaretIcon = ({
   fill,
   size = 'm',
   title = null,
-  direction = 'down'
+  direction = 'up'
```

Comment:
Was this deliberate? It looks like this icon is pointing the wrong way now 🤔

Reply:
No, and thank you for catching! There was some noise in the rebase as this was an old branch, and I missed this 👍